libibverbs/libibverbs.map.in

```diff
@@ -80,6 +80,7 @@ IBVERBS_1.1 {
 		ibv_get_device_guid;
 		ibv_get_device_list;
 		ibv_get_device_name;
+		ibv_get_pkey_index;
```

Comment:
This needs its own stanza:

```c
IBVERBS_1.5 {
       global:
                         ibv_get_pkey_index;
} IBVERBS_1.1;
```

Reply:
And the CMakeLists.txt needs to be modified:

```
rdma_library(ibverbs "${CMAKE_CURRENT_BINARY_DIR}/libibverbs.map"
  # See Documentation/versioning.md
  1 1.5.${PACKAGE_VERSION}
```

Reply:
An additional change was needed. Can you have another look? Thanks, Bart.